bin/commitments_deprecater.rb

```diff
@@ -0,0 +1,190 @@
+# frozen_string_literal: true
+
+require "cluster"
+require "services"
+require "deprecation_record"
+
+Services.mongo!
+
+# Takes (shared print) deprecation records, and tries to match them to commitments
+# so that those commitments can be deprecated.
+# Usage: CommitmentsDeprecater.new.run(<list_of_files>)
+
+class CommitmentsDeprecater
```

Comment:
We probably want to move this to `lib`, but I'm not sure what subdirectory to put it in (maybe a new `shared_print` directory?) This isn't exactly a loader like the others..

Reply:
addressed